public/swagger/cashmanager.yml

```diff
@@ -180,7 +180,171 @@ paths:
   # Regras da Category
```

Comment:
Faltando a tag no início, mas acho que você ainda vai terminar essa documentação, né

Ex:
```
tags:
  - name: auth
    description: Recursos relacionados ao usuário.
  - name: category
    description: Recursos bla blabla.

```